migrations/sql/V2019.01.17.12.10__create_minespace_user_tables.sql

```diff
@@ -11,12 +13,15 @@ CREATE TABLE minespace_user (
 
 COMMENT ON TABLE minespace_user IS 'A list of public mine proponents email addresses that can log into minespace and update mds information on mines listed in minespace_user_mds_mine_access';
 
-CREATE TABLE minespace_user_mds_mine_access(
+drop table if exists minespace_user_mds_mine_access;
+CREATE TABLE minespace_user_mds_mine_access
+(
     user_id integer NOT NULL,
-    mine_guid uuid NOT NULL, 
+    mine_guid uuid NOT NULL,
 
```

Comment:
Should the 4 audit columns be added to this table too? Also, is there a need to keep active and inactive records? What happens if the user is no longer associated with a mine, does it get deleted? If so, should it?

Reply:
@klefler, 
Same thing, didn't apply the pattern to this. i guess it's a question for the business if that information is required. 
Yes it currently hard deletes.